### ISSUE TITLE
Background attachment?

### ISSUE THREAD

**Issue by benface** [hacknug/tailwindcss-background-extended#1]
Tailwind itself already has a `backgroundAttachment` module since [v0.3.0](
https://github.com/tailwindcss/tailwindcss/releases/tag/v0.3.0), which generates the same classes as this plugin:

```
.bg-fixed {
  background-attachment: fixed;
}

.bg-local {
  background-attachment: local;
}

.bg-scroll {
  background-attachment: scroll;
}
```

Should these classes be removed from this plugin then?

**Comment by hacknug** [comment]
Thanks for the heads up, I probably missed it somehow. Will release a new version later and another one when/if Adam merges your PR with `space` and `round`.

**Comment by hacknug** [comment]
/remind me to update the plugin in 4 days.

**Comment by hacknug** [comment]
No news at https://github.com/tailwindcss/tailwindcss/pull/610 yet 😢

/remind me in a week

**Comment by benface** [comment]
👋 @hacknug, update the plugin .

-- benfacebot

**Comment by hacknug** [comment]
@benface I've just published v1.0.0 which removes support for everything we already have in core, adds tests (which I shamelessly stole from https://github.com/benface/tailwindcss-transforms) and introduces support for `backgroundImage` which I'm not sure is even needed 🤷‍♂

**Comment by benface** [comment]
Nice update! 👍
